doc/Makefile

```diff
@@ -1,7 +1,7 @@
 # Makefile for Sphinx documentation
 #
 
-PYVER = 3.6
+PYVER = 3.7
```

Comment:
This one should not be changed, 3.6 is widespread these days, will probably be supported for a couple of years, and so using it to build the documentation makes sense. OTOH, 3.7 is brand new and not yet included in many distros.

Reply:
So do we revert Circle CI to Py36?

Reply:
>So do we revert Circle CI to Py36?

Hmm, to tell the truth, I don't know ;) Did you try it with only one of the two changed? It cannot hurt to stick with 3.6 for a while.

Reply:
CircleCI change without the Makefile change is:
* https://circleci.com/gh/numpy/numpy/2660?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Reply:
Guess it matters, good to know.